openedx/features/learner_profile/static/learner_profile/templates/section_two.underscore

```diff
@@ -2,9 +2,9 @@
     <div class="field-container"></div>
     <% if (!showFullProfile) { %>
         <% if(ownProfile) { %>
-            <span class="profile-private--message" tabindex="0"><%- gettext("You are currently sharing a limited profile.") %></span>
```

Comment:
@HarryRein I do so little front-end.  Remind me why tabindex was here and why it is being removed?